### ISSUE TITLE
Loss of feature extractor training

### ISSUE THREAD

**Issue by shunlinlu** [EricGuo5513/text-to-motion#21]
Hi, authors

Thanks for presenting such great work. I followed your instruction to train the feature extractors, the training loss reduces normally, but the val loss rises after a few iterations. The encodes seem to be OVERFITTING heavily. I use your pretrianed vae checkpoint and the only change is the batch size to 128. 
Here is my log. 

[train_text_mot_match_humanml.log](https://github.com/EricGuo5513/text-to-motion/files/10840000/train_text_mot_match_humanml.log)

**Comment by EricGuo5513** [comment]
Hi, this happened to me as well. I use the checkpoint that has the minimum
validation loss.
